cluster/config-defaults.yaml

```diff
@@ -374,9 +374,16 @@ skipper_open_policy_agent_enabled: "false"
 skipper_open_policy_agent_styra_token: ""
 skipper_open_policy_agent_data_preprocessing_optimization_enabled: "false"
 
+
 # Default timeout value in seconds for outgoing http calls from Open Policy Agent in a skipper filter
 skipper_open_policy_agent_styra_response_header_timeout: "2"
 
+# Decision logging use event buffer type
+skipper_open_policy_agent_decision_logs_use_buffer_type_event: "false"
+
+# Decision logging sets the maximum number of decision log events that can be buffered before being dropped
+skipper_open_policy_agent_decision_logs_buffer_size_limit_events: "1000"
```

Comment:
-> `skipper_open_policy_agent_decision_logs_buffer_type_event_limit`

Keys of similar things should form a group.